src/doc/src/reference/manifest.md

```diff
@@ -258,8 +258,8 @@ The `license` field contains the name of the software license that the package
 is released under. The `license-file` field contains the path to a file
 containing the text of the license (relative to this `Cargo.toml`).
 
-[crates.io] interprets the `license` field as an [SPDX 2.1 license
-expression][spdx-2.1-license-expressions]. The name must be a known license
+[crates.io] interprets the `license` field as an [SPDX 2.3 license
+expression][spdx-2.3-license-expressions]. The name must be a known license
 from the [SPDX license list 3.11][spdx-license-list-3.11]. Parentheses are not
```

Comment:
It seems that crates.io now use SPDX license list v3.20. could you also update this to 3.20?

<https://github.com/spdx/license-list-data/tree/v3.20>